docs/artifacts/thirty-eight-tools.html

```
@@ -0,0 +1,397 @@
+<title>Thirty-Eight Tools</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)